### DISCUSSION TITLE
E2E 密钥错误, 无法解密对端握手请求: Error: wrong secret key for the given ciphertext

### DISCUSSION THREAD

**Discussion by tencentbook** [natfrp/wiki#464] [使用问题] [answered]
E2E 密钥错误, 无法解密对端握手请求: Error: wrong secret key for the given ciphertext，2024.4.20一整天都无法登陆了

**Comment by fakeboboliu** [comment]
通常可能是因为您的密码有误，更特别的可能是什么东西将您客户端的配置文件修改了

**Reply by tencentbook** [reply]
4.19还能正常连接，然后没有调整任何东西，4.20突然连接不了，4.21依旧无法连接，我新建了一个服务器重新安装了一个依然无法连接

**Reply by tencentbook** [reply]
我当前内网三台设备，一台设备a内网穿透功能正常但无法连上远程管理，一台设备b也是内网穿透功能正常但无法连上远程管理反复实验也无法生效，于是我4.20卸载了b设备上的樱花软件在c设备上从零开始安装，但换了3次远程访问的密码知道4.21均无法连上远程管理。三台设备是群晖ds420+和群晖ds920+和centos7.9

[a comment by an automated account is not shown]

**Comment by tencentbook** [comment]
2024.4.21-下午1点后我重启服务器和重启服务几次后可以访问了

**Comment by tencentbook** [comment]
2024.4.21-下午7点后又不能正常访问，我重启服务器和重启服务几次后内网穿透直接中断，远程管理和内网穿透均不能正常访问

**Reply by fakeboboliu** [reply]
就您的说明来看，您可以尝试检查一下网络、审查及服务是否运行

远程管理的中心服务器与内网穿透的节点差别很大，如果它们同时不工作，这可能表征您的网络存在问题或者它们实际上没有在运行

**Reply by tencentbook** [reply, marked as answer]
2024.4.28和2024.4.30 对比验证，在没有任何改动的情况下可以正常连接，应该是电信网络和樱花穿透网络存在问题，可能是被恶意封锁也可能仅仅是网络波动
